D1673/P1673.bs

```diff
@@ -1681,14 +1679,21 @@ that create significant temporary storage
 only if the value type is trivially constructible,
 trivially destructible, and not too large.
 
-Regarding Assumption (4):
+Regarding Assumption (2):
 The freedom to compute sums in any order is not necessarily a type constraint.
 Rather, it's a right that the algorithm claims,
-regardless of whether the type is actually associative or commutative. 
+regardless of whether the type's addition is associative or commutative.
 For example, floating-point sums are not associative,
 yet both parallelization and customary linear algebra optimizations
 rely on reordering sums.
-See the above "adverbs, not adjectives" section for a more detailed explanation.
+See the above "Value type constraints do not suffice to describe algorithm behavior" section
+for a more detailed explanation.
+
+Regarding Assumption (3),
+we do not actually say that
+value-initialization produces a two-sided additive identity.
+What matters is what the algorithm's implementation may assume,
```

Comment:
Using "assume" here isn't as much of a problem as it would be in the wording, but "do" would work just as well.

Reply:
Fixed; thanks!